src/ocean/global.F90

```diff
@@ -138,10 +145,6 @@ subroutine initialize(a_n,a_m,a_l,&
 
     ! if (n/=a_n .or. m/=a_m .or. l/=a_l) then
```

Comment:
Seems like this comment should go to. Apparently there were hardcoded N,M,L in some old version?